examples/powercli/eventbridge/handler/script.ps1

```diff
@@ -0,0 +1,32 @@
+# Process function Secrets passed in
+$SECRETS_FILE = "/var/openfaas/secrets/eventbridge-secrets"
+$SECRETS_CONFIG = (Get-Content -Raw -Path $SECRETS_FILE | ConvertFrom-Json)
+
+# Process payload sent from vCenter Server Event
+$json = $args | ConvertFrom-Json
+if($env:function_debug -eq "true") {
+    Write-Host "DEBUG: json=`"$($json | Format-List | Out-String)`""
+}
+
+Import-Module AWS.Tools.EventBridge
+
+$details = [pscustomobject] @{
+    CreatedTime = $json.CreatedTime;
+    UserName = $json.UserName;
+    VMName = $json.objectName;
+}
+
+$data = ($details | convertTo-Json).toString()
+
+$payload = New-Object Amazon.EventBridge.Model.PutEventsRequestEntry
+$payload.EventBusName = "VMware-VMC" #EventBride Bus Name
```

Comment:
Why not make this an ENV so we don't hardcode/assume the bus name?

Reply:
Good point! Fixed and pushed the changes already